doc/contents/LoadMagicCommand.ipynb

```diff
@@ -0,0 +1,87 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%classpath add jar resources/jar/loadMagicJarDemo.jar"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Source code of ShowEvnsCustomMagicCommand class\n",
```

Comment:
link to the file on github instead of copying it here

Reply:
We will do it after merging branch to the master, because no such file in master.